Target-MVD-Mathias/Views/ChatTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ChatTableViewCell.swift
+//  Target-MVD-Mathias
+//
+//  Created by Mathias Cabano on 9/26/17.
+//  Copyright © 2017 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class ChatTableViewCell: UITableViewCell {
+  
+  // MARK: Outlets
+  @IBOutlet weak var userAvatarImageView: UIImageView!
+  @IBOutlet weak var userNameLabel: UILabel!
+  @IBOutlet weak var lastMessageLabel: UILabel!
+  @IBOutlet weak var chatTopicImageView: UIImageView!
+  @IBOutlet weak var unreadMessagesView: UIView!
+  @IBOutlet weak var unreadMessagesLabel: UILabel!
+  
+  // MARK: Setup
+  func setup(withMatch match: Match) {
+    setupAvatarImageView(image: match.user.image)
+    
+    userNameLabel.text = match.user.username
+    lastMessageLabel.text = match.lastMessages.last ?? "This conversation hasn't started yet"
+    
+    chatTopicImageView.sd_setImage(with: match.topic.icon)
+    chatTopicImageView.alpha = match.unread > 0 ? 1 : 0.5
+    
+    setupUnreadMessagesView(withUnreadMessages: match.unread)
+  }
+  
+  private func setupAvatarImageView(image: URL?) {
+    userAvatarImageView.layer.cornerRadius = userAvatarImageView.frame.size.width/2
```

Comment:
WDYT about spaces between operators like this `userAvatarImageView.frame.size.width / 2`?
I think it makes it look cleaner. Same for the method above.

Reply:
I use spaces for + and - operators, but not for / and *, but I can change that :)